integrations/purchase/bridge/bridge-api-categories.js

```diff
@@ -208,3 +208,38 @@ export const BRIDGE_API_CATEGORIES = Object.entries(idToCategory).reduce(
   },
   {}
 );
+
+// Deliberately want activities that match the following
+// To be created with the ignored=true value
+export const BRIDGE_API_CATEGORIES_ID_TO_IGNORE = [
```

Comment:
To have a common interface, it would be nice to use the same name here for different integrations.

What about `CATEGORY_IDS_TO_IGNORE`?

Reply:
Just a thought here, as it might also help prepare https://github.com/tmrowco/bloom/issues/97:
Why not expose a method that takes a bridge category, amount and currency and returns an activity? That way we can have, in one go, the ability to set the ignore field, the activitytype field, and the lineitem field?

Food for thought. Feel free to ignore if irrelevant.

Reply:
Thanks for the thought Oli, nevertheless, I think that it exceeds the scope of the task here so I won't implement it but will note it in the other issue :) 

regarding the naming, I ended up going for `CATEGORIES_TO_IGNORE` as it won't always be IDs